readme.md

```diff
@@ -26,7 +27,7 @@ answer goes here
 What is the difference between a fork and a clone?
 
 ```
-answer goes here
+a fork copy's the homework into your github account, and a clone copies the homework to the computer (so you will see the folder show up in your computer files). 
 
 ```
 
```

Comment:
👍 I'll accept this, but forking technically creates a copy of another repo hosted on GitHub to your GitHub account.